internal/pkg/cli/env_init.go

```diff
@@ -48,11 +49,53 @@ var (
 	errNamedProfilesNotFound = fmt.Errorf("no named AWS profiles found, run %s first please", color.HighlightCode("aws configure"))
 )
 
+type importVPCVars struct {
+	ID               string
+	PublicSubnetIDs  []string
+	PrivateSubnetIDs []string
+}
+
+func (v importVPCVars) isSet() bool {
+	if v.ID != "" {
+		return true
+	}
+	if len(v.PublicSubnetIDs) != 0 {
+		return true
+	}
+	if len(v.PrivateSubnetIDs) != 0 {
+		return true
+	}
+	return false
+}
+
+type adjustVPCVars struct {
+	CIDR               net.IPNet
+	PublicSubnetCIDRs  []string
+	PrivateSubnetCIDRs []string
+}
+
+func (v adjustVPCVars) isSet() bool {
+	if v.CIDR.String() != deploy.EmptyIPNetString {
```

Comment:
Since the const is only used in this package, should it be defined here instead of `deploy`?
or can we do:
```go
if v.CIDR == net.IPNet{}
```

Reply:
`if v.CIDR == net.IPNet{}` would say "==" is not defined...

Reply:
cool cool, then maybe this: https://play.golang.org/p/scc2htuBHor

Reply:
ok addressed!